data/human/intro missions.txt

```diff
@@ -413,7 +413,100 @@ mission "Intro [2 Interceptor]"
 				`	"Sure, but I can't wait until I've got a ship big enough to stand up to pirates instead of running from them."`
 				`	"Well, maybe I should've bought a freighter instead."`
 			`	"Better start saving up then," he says. "I know a number of old Sparrow pilots who made their first million credits mining asteroids before they could afford a more combat-worthy ship. They had to look around for a while before finding a system with any asteroids worth mining, but it was worth it. Anyway, let's go get your payment." You walk out of the ship and sign the papers to hand off the <commodity> to the dock workers on <planet>. They hand you a credit chip worth <payment>.`
-			`	"If you're willing to take me on one more journey," says James, "meet me in the spaceport bar in a little while."`
+			`	"I've got an idea for something you might want to try," says James. "Meet me in the spaceport bar in a little while."`
+
+
+
+mission "Intro [2.1 Interceptor]"
+	priority
+	name "Mine at <planet>"
```

Comment:
```suggestion
	name "Go to the <system> system and mine 5 tons of Silicon, then land on <planet>."
```

Reply:
This is the name field, not the description field.

Reply:
```suggestion
	name "Mine asteroids in <system>"
```